src/array.vala

```diff
@@ -2,6 +2,9 @@ using GLib;
 
 public class Vast.Array : Object
 {
+    public const ssize_t NEW_AXIS = ssize_t.MAX;
+    public const size_t DEFAULT_DIMENSION = size_t.MAX;
```

Comment:
We may need to keep track of some of the `axis` changes to ensure no original axes is used twice, because

the current way we can end up with broken strides and shape. e.g.

```
.axis(0, 0)
.axis(1, 0)
```